data/crawler/src/processing/manager.py

```
@@ -0,0 +1,163 @@
+
+        if self.executor:
+            logger.info("Waiting for pending tasks...")
+
+            with self.lock:
+                futures = [f for f, _ in self.processing_futures]
+
+            for future in as_completed(futures):
+                try:
+                    future.result()
+                except Exception:
+                    pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)